src/flamegraph/mod.rs

```diff
@@ -317,7 +323,7 @@ where
     W: Write,
 {
     let mut reversed = StrStack::new();
-    let (mut frames, time, ignored, delta_max) = if opt.reverse_stack_order {
+    let (mut frames, time, ignored, delta_max) = if opt.reverse_stack_order || opt.flame_chart {
```

Comment:
I'm not sure I follow why flame chart mode should use reverse stack order?

Reply:
Looks like I confused reverse stack order with reverse timeline.